sports.html

```diff
@@ -253,14 +253,15 @@ <h3><a href="https://punchng.com/wcup-qualifier-slimani-scores-as-algeria-beat-c
             </section>
 
 
+            <hr class="horizontal" />
             <section class="section-news-content-container">
                 <!--one-->
                 
                 <div class="content-container">
                   <a
                     href="https://punchng.com/head-to-head-how-nigeria-lost-25-of-56-previous-encounters-against-ghana/"
                   >
-                    <img src="./Eagles-training1.webp" alt="image" />
```

Comment:
Please copy the image address. You doesn't need to download the image. Do well to effect the correction.
Thank you